collection/collection/src/commonMain/kotlin/androidx/collection/SimpleArrayMap.kt

```diff
@@ -288,17 +288,19 @@ public open class SimpleArrayMap<K, V> @JvmOverloads public constructor(capacity
     // Java's Map interface, which is necessary since ArrayMap is written in Java and implements
     // both Map and SimpleArrayMap.
     public open fun getOrDefault(key: Any?, defaultValue: V): V {
-        return getOrDefaultInternal(key, defaultValue)
+        return getOrDefaultInternal(key, defaultValue) as V
     }
 
     @Suppress("NOTHING_TO_INLINE")
-    private inline fun <T : V?> getOrDefaultInternal(key: Any?, defaultValue: T): T {
+    // TODO: revert the change: this function was changed in JB fork because of https://youtrack.jetbrains.com/issue/KT-65061
+    private inline fun getOrDefaultInternal(key: Any?, defaultValue: Any?): Any? {
```

Comment:
Should we really change prod/common code because of issues in web tests?

Reply:
It's not in the tests only. The real code 
in the apps also affected.
It's not the first case where this issue has surfaced and this workaround was already applied in other places.